boards/arm/mimxrt685_evk/pinmux.c

```diff
@@ -278,6 +278,126 @@ static int mimxrt685_evk_pinmux_init(const struct device *dev)
 	IOPCTL_PinMuxSet(IOPCTL, 1U, 6U, port1_pin6_config);
 #endif
 
+#if (DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(flexcomm1), nxp_lpc_i2s, okay)) && \
+	(DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(flexcomm3), nxp_lpc_i2s, okay)) && \
```

Comment:
Why use flexcomm1 in the first line and flexcomm3 in the second line? I assume this is a typo?

Reply:
I have updated this per feedback from @galak.

Reply:
I see what this is doing now, this is needed for the shared signals. That said, can you pull the pinmuxing out of this conditional and use separate conditionals for each instance? The way it's configured now won't work if you only enable one instance.

Reply:
For the mimxrt685 EVK board we are using both TX and RX I2S. The flexibility to have only one enabled is provided in the common files such as I2S and soc.c